duckduckgo/iOS
# Update about screen

Task/Issue URL: https://app.asana.com/0/414709148257752/1205916913109009/f
Tech Design URL:
CC:

**Description**:

Convert about screen to SwiftUI and update content. 

**Steps to test this PR**:
1. Launch the app on iOS 15, 16, 17 phone and iPad and check about screen looks OK.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
